pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -77,11 +78,11 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 	logger := logging.FromContext(ctx)
 	logger.Info("Let us do a reconcile.")
 
-	// Get the Route resource with this namespace/name.
+	// Get the KnativeServing resource with this namespace/name.
 	original, err := c.knativeServingLister.KnativeServings(namespace).Get(name)
 	if apierrs.IsNotFound(err) {
 		// The resource may no longer exist, in which case we stop processing.
-		logger.Errorf("route %q in work queue no longer exists", key)
+		logger.Errorf("KnativeServing %q in work queue no longer exists", key)
 		return nil
```

Comment:
This is the point at which you should invoke `DeleteAll` on the manifest to clean up the installed resources, I think.

Reply:
Is this used to resolved the installation of all the knative resources?

Reply:
I'm not sure I understand your question, but I assume `Reconcile` gets called when some lifecycle event occurs for the KnativeServing instance (or one of its children), so if it's not found, logically that means it's been deleted, correct? At which point, we should clean up its resources.